_plays/08.md

```diff
@@ -4,10 +4,12 @@ title: Make it easy to use
 ---
 
 ### Checklist
-1.  Provide reference implementation.  Show how to use the API.
-2.  Write modifiable exampls.
-3.  Even create cliend-side wrappers with use, if you have the resources.  This brings the service even closer to the end-use.
+1.  Provide reference implementation.  
+2.  Write modifiable examples.
+3.  Even create client-side wrappers with use, if you have the resources.  This brings the service even closer to the end-use.
 4.  Try it out yourself!
 
 ### Key Questions
-- 
\ No newline at end of file
+- How long does it take to build a toy application on the service?
+- Can the service be used in a hackathon?  Even if you are not there to guide or instruct participants?
+- Is there a way to easily incorporate the underlying data into a rectangular data format for statistical analysis or reporting?
```

Comment:
Like a CSV.  It would be an exercise in distilling data into a form that statisticians or econometricians would use.  If this is super hard to do, or at least super hard to instruct another developer how to do, then something is wrong.  So for example, showing how to use the OSM web service to built a table of water fountains by census block.